src/content/libs/utils.js

```diff
@@ -22,3 +23,13 @@ export const notifyIf = async (option, message) => {
     })
   }
 }
+
+export const isLoggedIn = () => {
+  const mainHeaderRightLoggedOut = select('.main-header__right__logged-out')
```

Comment:
`select-dom` has a function called `select.exist` which we could use instead: https://www.npmjs.com/package/select-dom#selectexistsselector-parent--document